spec_util/meld.go

```diff
@@ -674,6 +697,11 @@ func (m *melder) meldOneOfVariant(dst *pb.OneOf, srcHash *string, srcVariant *pb
 		srcHash = nil
 	}
 
+	// Clear the nullable bit from srcVariant.  The nullable bit was
+	// copied to the dst Data object in meldData before calling
+	// meldOneOfVariant.
+	srcVariant.Nullable = false
```

Comment:
The block above makes a copy of `srcVariant` before modifying it, and ensures that the block below rehashes the modified variant. Shouldn't we do something similar here?

Reply:
Fixed in f90bb9b14868c883e1a2e71989fff523318e4939.